### ISSUE TITLE
Status page: Indicate whether there are enabled modules with `schema_version` of `-1`

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#6645]
This is a thing that shouldn't happen under normal circumstances, however, as discussed in #6438 and in [this Zulip thread](https://backdrop.zulipchat.com/#narrow/stream/218635-Backdrop/topic/Database.20updates.20-.20out.20of.20date.20-.20finding.20the.20cause), it does happen (and has some odd UX side effects when it does).

I believe that it would be helpful to at least point out in the site status report when this happens, and be listing the modules that get into that situation. Perhaps if people start reporting these problems, we might be able to see a pattern and get to the bottom of why it happens.

**Comment by nattywebdev** [comment]
Thanks for recording this. It's a real problem that I encounter but only when upgrading from D7 to B, and always on only a few contrib modules. No particular pattern that I can discern, as it's rarely the same modules that cause it in my experience. Next time I have it I will record it here.

**Comment by bugfolder** [comment]
I agree with this check. It was puzzling when it happened to a site I was doing, and it's regularly puzzling to people who report it in Zulip. One wonders how many non-Zulip devs just ignore it as "one of those puzzling things but the site works fine otherwise."
